olive/passes/onnx/conversion.py

```diff
@@ -119,6 +120,9 @@ def _initialize(self):
     def _run_for_config(self, model: PyTorchModel, config: Dict[str, Any], output_model_path: str) -> ONNXModel:
         pytorch_model = model.load_model()
         pytorch_model.eval()
+        # put pytorch_model and dummy_inputs at the same device
+        torch_device = pytorch_model.device
```

Comment:
seems the pipeline fail here since there is no device attribute on model?

Reply:
https://stackoverflow.com/questions/58926054/how-to-get-the-device-type-of-a-pytorch-module-conveniently

Look like it is impossible to get device of model, I just put model and data to cpu direcly.

Reply:
sure